### DISCUSSION TITLE
Translate

### DISCUSSION THREAD

**Discussion by ridvanbesli** [IBAX-io/go-ibax#756] [Ideas]
Programın daha iyi anlaşılabilmesi amacıyla Türkçe diline uyumlu bir çeviri sistemi oluşturulabilir mi?

**Comment by ylcndgn** [comment]
Ibax coinlerimi goremiyorum. Cuzdanima nasil ulasabilirim

25 Oca 2022 Sal 17:45 tarihinde ridvanbesli ***@***.***> şunu
yazdı:

**Reply by ridvanbesli** [reply]
Puanlarını mı coinlerini mi?

**Comment by ylcndgn** [comment]
Bu bolume ulaşamıyorum

25 Oca 2022 Sal 17:47 tarihinde Yalçın DOĞAN ***@***.***> şunu
yazdı:

**Comment by ylcndgn** [comment]
Coinleri

25 Oca 2022 Sal 17:57 tarihinde ridvanbesli ***@***.***> şunu
yazdı:
